### PR TITLE
test(sdk) Update the Query Editor Button

### LINKED ISSUE #46096: Make Query Editor Button Bigger
This is the most important button on the screen, it shouldn't be the smallest

Screen Shot 2024-07-24 at 2 26 55 PM

this is gonna break a lot of tests

### PR DESCRIPTION
**Pull request by iethree**
Closes https://github.com/metabase/metabase/issues/46096

### Description

Before | After
---|---
Screen Shot 2024-07-24 at 2 37 33 PM | Screen Shot 2024-08-02 at 1 39 13 PM

notebook side:
Screen Shot 2024-08-02 at 1 39 22 PM

qb header update

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionNotebookButton/QuestionNotebookButton.tsx
Comment (iethree) [inline frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionNotebookButton/QuestionNotebookButton.tsx:1]: this is the only non-test file changed

File: frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionNotebookButton/QuestionNotebookButton.tsx [hunk 1]
```diff
@@ -21,22 +21,15 @@ export function QuestionNotebookButton({
...
-    <Tooltip
-      label={isShowingNotebook ? t`Hide editor` : t`Show editor`}
-      data-placement="top"
-      position="top"
+    <Button
+      data-testid="notebook-button"
+      onClick={() =>
+        setQueryBuilderMode(isShowingNotebook ? "view" : "notebook")
+      }
+      // leftIcon={<Icon size={14} name="notebook" />}
```
Comment (sloansparger) [inline frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionNotebookButton/QuestionNotebookButton.tsx:29]: not sure if this comment is needed

**Review by sloansparger — APPROVED** [review]

**Review by mazameli — CHANGES_REQUESTED** [review]
1. I'd love it if these buttons could all look the same. The Show Editor button's height and borders are currently different.

image

2. I don't love that the Show Visualization button is present from a totally new GUI question, and would prefer that it only appears for saved questions, or for cases where you've arrived in the editor via the Show Editor button. If that's unavoidable or costlier than we're willing to spend right now, I think I can live with it.

**Comment by iethree** [comment]
> 1. I'd love it if these buttons could all look the same. The Show Editor button's height and borders are currently different.

I have now rebased this branch off of master to pull in recent changes to those buttons, they should now be perfectly identical in this branch as well.

Screen Shot 2024-07-29 at 3 45 35 PM

> 2. I don't love that the Show Visualization button is present from a totally new GUI question, and would prefer that it only appears for saved questions, or for cases where you've arrived in the editor via the Show Editor button. If that's unavoidable or costlier than we're willing to spend right now, I think I can live with it.

I will make this change in another branch since it's essentially a new feature: context-sensitive notebook button, and not just a simple visual change.

**Review by mazameli — CHANGES_REQUESTED** [review]
I'm on `771855b792b2e6ee178a822dc146758ec22706ae` and I'm not seeing the behavior from https://github.com/metabase/metabase/pull/46350 incorporated in this branch. 

https://www.loom.com/share/fe34aa25597f455bb1d061d45f190541

**Review by mazameli — APPROVED** [review]
Works now 🤷🏼‍♂️

**Pull request merged**